configs/_base_/default_runtime.py

```diff
@@ -19,3 +19,8 @@
 opencv_num_threads = 0
 # set multi-process start method as `fork` to speed up the training
 mp_start_method = 'fork'
+
+# In order to support automatically scaling LR. Default batch size
+# (8 GPUs) x (2 samples per GPU) or (4 GPUs) x (4 samples per GPU)
+default_batch_size = 16
+enable_auto_scale_lr = False
```

Comment:
We can set `default_batch_size = None` and let users decide the batch size of their models.

Reply:
According to the mmdet original doc [train-predefined-models-on-standard-datasets](https://github.com/open-mmlab/mmdetection/blob/master/docs/en/1_exist_data_model.md#train-predefined-models-on-standard-datasets), It seems that they already modefided the LR to batch size 16. Cuz if there are multi version of batch size of different model , the user will be confused.

What's more if the user need to change the LR according to the metafile.yml, the doc will show them how to do, but I can't find any doc about the metafile.yml, so I think this default batch size of mmdet is all 16.

In conclusion, I think we can set it to 16.

Reply:
Cuz if it set to `None`, I think is half-auto not full auto. Pls consider it.

Reply:
We can set it as 16, and users can modify their config if they need to change it.